src/main/java/jmgomez/apipolicy/controller/PolicyController.java

```diff
@@ -8,30 +8,32 @@
 import org.springframework.security.core.context.SecurityContextHolder;
 import org.springframework.web.bind.annotation.*;
 
+import java.util.List;
+
 @RestController
 public class PolicyController {
 
     @Autowired
     private PolicyService policyService;
 
     @GetMapping("/policies")
-    public PolicyDto[] getPolicies() {
+    public List<PolicyDto> getPolicies() {
         return policyService.getPolicies(getUserId());
     }
 
     @GetMapping("/policies/{id}")
-    public PolicyDtoCov getPolicyByIDs(@PathVariable String id) {
-        return policyService.getPoliciesByIDs(getUserId());
+    public PolicyDtoCov getPolicyByIDs(@PathVariable("id") String id) {
```

Comment:
PolicyDtoCov? What's that? The multiple Policy DTOs are very confusing. Also just noticed our API is exposing models in spanish... we use English in our models, if the endpoints we have as source are in spanish, that's a projection task we need to take care of as part of our standardization. But we shouldn't expose models in spanish to our apps.